ruby-challenges.rb

```diff
@@ -30,3 +75,41 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+# psuedo code
+# add a new method for pedal_faster
+# use += to add to the current speed
+# add a new method for braking 
+# make the speed decrease 
+# make the current speed 0 if @currentspeed is less than 0 since we cannot go negative 
+class Bike
+    def initialize(model, wheels = 2, current_speed = 0)
+      @model = model
+      @wheels = wheels
+      @current_speed = current_speed
+    end
+  
+    def bike_info
+      "The #{@model} bike has #{@wheels} wheels and is going #{@current_speed} mph."
+    end
+
+    def pedal_faster(speed_increase)
+    @current_speed += speed_increase
+    end
+
+    def brake(speed_decrease)
+    @current_speed -= speed_decrease
+    @current_speed = 0 if @current_speed < 0
+    end
+end
+    
+my_bike = Bike.new('bike ')
+p my_bike.pedal_faster(10)
+p my_bike.pedal_faster(18) 
+p my_bike.brake(5) 
+p my_bike.brake(25) 
+
+# 10
+# 28
+# nil not sure why I got nil here for the output 
```

Comment:
You're getting nil because you are printing the output of `my_bike.brake(number)`. If you printed `my_bike.bike_info` instead, you would see the intended output. Nice work!